source/simulator/helper_functions.cc

```diff
@@ -1814,9 +1812,7 @@ namespace aspect
 
     initialize_current_linearization_point();
 
-    pcout << "in "
-          << number_of_reaction_steps
-          << " substep(s)."
```

Comment:
I don't recall if there was a way to ask the ARKode object for the number of steps it took. But if there isn't, then this will work:
```
unsigned int n_ode_timesteps = 0;
ode.output_step ([&n_ode_timesteps](const double       t,
                       const VectorType & sol,
                       const unsigned int step_number) {
  n_ode_timesteps = step_number;
});

Reply:
I checked, and the `solve_ode` function returns the final number of computed steps. So I could use that, but it might be a different number for each cell. So the question is: What would be a reasonable number to return? The average? The maximum? The sum?